auth_app/models.py

```diff
@@ -35,3 +39,24 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return "@{}".format(self.username)
+
+
+@receiver(post_save, sender=User)
+def after_user_signed_up(sender, instance, **kwargs):
+    if Bot.objects.filter(user=instance,).count() == 0:
```

Comment:
nit: you can use `if Bot.objects.filter(user=instance,).exists()`

Reply:
I tried with `exists` but it doesn't create my bot.